moc_openstack_api_reporting.py

```diff
@@ -734,7 +734,7 @@ def main():
 
     vol_keys = list(cluster_state["vol_timestamps"].keys())
     for key in vol_keys:
```

Comment:
This is unrelated to this PR and I'm not asking you to make this change. Just noting this as an opportunity to write better Python code in the future.

In Python 3, you can iterate through keys in a dictionary just by doing `for key in cluster_state["vol_timestamps"]`. Turning it into a list is not necessary.

Reply:
I think it's because he's deleting the list he's iterating over, so that's why he wanted a copy of just the keys.

Reply:
Good point, thanks. I missed that.

Reply:
@naved001 is absolutely correct.  Since we are deleting keys we need a copy that is not changing.